libexec/rtld-elf/Symbol-c18n.map

```diff
@@ -11,4 +11,9 @@ FBSDprivate_1.0 {
     _rtld_longjmp;
     _rtld_safebox_code;
     _rtld_sandbox_code;
+    _rtld_unw_getcontext;
+    _rtld_unw_setcontext;
+    _rtld_unw_getcontext_unsealed;
+    _rtld_unw_setcontext_unsealed;
+    _rtld_unw_getsealer;
```

Comment:
Better to group them under `_rtld_setjmp` and `_rtld_longjmp`. So something like
```
_rtld_setjmp
_rtld_unw_getcontext
_rtld_unw_getcontext_unsealed
_rtld_longjmp
_rtld_unw_setcontext
_rtld_unw_setcontext_unsealed
_rtld_unw_getsealer
```